src/calendar-v1/stories/Calendar.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import { CalendarNextYearButton } from "../CalendarNextYearButton";
+import { CalendarNextMonthButton } from "../CalendarNextMonthButton";
+import { CalendarPreviousYearButton } from "../CalendarPreviousYearButton";
+import { CalendarPreviousMonthButton } from "../CalendarPreviousMonthButton";
+import { addDays, addWeeks, subWeeks } from "date-fns";
+
+export default {
+  title: "Component/Calendar",
+} as Meta;
+
+const Calendar: React.FC<CalendarProps> = props => {
+  const state = useCalendarState(props);
+  console.log("%c state", "color: #e5de73", state);
+
+  return (
+    <div className="calendar">
+      <div className="header">
+        <CalendarPreviousYearButton {...state} className="prev-year">
+          <svg
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="2"
+              d="M11 19l-7-7 7-7m8 14l-7-7 7-7"
+            ></path>
+          </svg>
+        </CalendarPreviousYearButton>
+        <CalendarPreviousMonthButton {...state} className="prev-month">
+          <svg
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="2"
+              d="M15 19l-7-7 7-7"
+            ></path>
+          </svg>
+        </CalendarPreviousMonthButton>
+        <h2 id="cb-grid-label" className="month-year" aria-live="polite">
+          {useDateFormatter({ month: "long", year: "numeric" }).format(
+            state.currentMonth,
+          )}
+        </h2>
+        <CalendarNextMonthButton {...state} className="next-month">
+          <svg
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="2"
+              d="M9 5l7 7-7 7"
+            ></path>
+          </svg>
+        </CalendarNextMonthButton>
+        <CalendarNextYearButton {...state} className="next-year">
+          <svg
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="2"
+              d="M13 5l7 7-7 7M5 5l7 7-7 7"
+            ></path>
+          </svg>
+        </CalendarNextYearButton>
+      </div>
+      <CalendarGrid {...state} as="table" className="dates">
+        <thead>
+          <tr>
+            <th scope="col" abbr="Sunday">
+              Su
+            </th>
```

Comment:
just adding a remainder here for improving the the screen reader announcing the weekdays.